practice/src/main/java/Main.java

```diff
@@ -1,5 +1,3 @@
-package code.practice;
```

Comment:
Please create package `practice` in `java` folder and move all the code inside this package. Don't forget to do the same for tests.